crates/napi/src/status.rs

```diff
@@ -30,6 +30,8 @@ pub enum Status {
   DetachableArraybufferExpected,
   WouldDeadlock,
   NoExternalBuffersAllowed,
+  /// The napi_env passed into the call was not valid
+  EnvInvalid,
```

Comment:
Don't need an additional error code here, these statuses are aligned to https://nodejs.org/api/n-api.html#napi_status